src/datePickerAndroid.rei

```diff
@@ -0,0 +1,24 @@
+[@bs.scope "DatePickerAndroid"] [@bs.module "react-native"] external dateSet : string =
```

Comment:
The convention in this database is to put externals as let bindings in `rei`s. So in this case it'd be `let dateSet : string;` etc.